tests/stack_test.go

```diff
@@ -811,3 +811,41 @@ func TestStackTags(t *testing.T) {
 	tags = lsTags()
 	assert.Equal(t, "projectValue", tags["projectTag"], "projectTag should be set to projectValue")
 }
+
+//nolint:paralleltest // pulumi new is not parallel safe
+func TestNewStackConflictingOrg(t *testing.T) {
+	t.Parallel()
```

Comment:
Should it be parallel or not? Hard to tell with the nolint and t.Parallel() call